packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,155 @@
+
+When single expand mode is enabled, the first accordion item is initially open, and can't be closed by clicking on it- it is only closed by clicking a different accordion item. At least one accordion has to be open at all times, which could be confusing to users as the expected functionality would be that all accordion items are initially closed until you open them and can be closed by clicking on that specific accordion item.
+
+This issue will be discussed with UX designers to determine if having this `single` expand mode functionality is necessary.
+
+Additionally, when using nested accordions with the single expand mode, once you open a child accordion, the parent accordion closes- when you open the parent accordion again, the previously selected child accordion is open, but when clicking on it, it doesn't close. Also, when opening the second child accordion, the first child accordion stays open, which goes against the functionality of single expand mode.
+
+This issue with nested accordions will be filed to FAST. Once FAST resolves the issue and UX confirms the `single` expand mode behavior, the `expand-mode` attribute will be updated to include the `single` expand mode.
+
+#### Nimble Accordion Item
+
+[FAST accordion-item API](https://github.com/microsoft/fast/tree/57f3c22c6341d8a21d48b1ffb7fcbfab1ffd02d8/packages/web-components/fast-foundation/src/accordion-item)
+
+-   _Component Name:_ `nimble-accordion-item`
+-   _Properties/Attributes:_
+    -   `error-visible` - boolean
+-   _Methods:_ Unchanged
+-   _Events:_ Unchanged
+-   _CSS Classes and Custom Properties that affect the component:_ Unchanged
+-   _Slots:_ The `start` and `end` slots will not be used. The `collapsed-icon` and `expanded-icon` slots are set by the nimble-accordion-item in its index.ts, so overriding them will not be supported.
+
+The design also includes an `error` state. This will be specific to each accordion item and controlled with the boolean attribute `error-visible`, which has a default attribute value of "". When needed, "error-visible" would be added to the accordion-item attributes, changing the color of the accordion item border color to red. This will be implemented in the accordion styling through conditional css styles based on `error-visible`'s value (ex. `:host([error-visible])` will have styling for when `error-visible` is true).
+
+Usage guidance for the `error-visible` state will be created, as its usage may be unclear.
+
+The shoelace design system switched to using `summary` and `details` elements in their accordion styled component, which allowed the [native browser search feature](https://github.com/shoelace-style/shoelace/pull/1470) to search inside of accordions that are closed. FAST does not allow this, so their template for the accordion item will be forked and changed to allow the native browser search feature.
```

Comment:
Can you provide some more details here on what the new template will look like and how significant the changes are from what FAST has? If you have prototyped this working, it would be helpful to put the template code from it here to give everyone an idea of what it'll look like.